### ISSUE TITLE
mMBART 50 finetuned many-to-one download link giving 403

### ISSUE THREAD

**Issue by patil-suraj** [pytorch/fairseq#3145]
Getting HTTP 403 access denied when downloading the mMBART 50 finetuned many-to-one model using the link in the [readme](https://github.com/pytorch/fairseq/tree/master/examples/multilingual#mbart50-models)

Here's the download link
https://dl.fbaipublicfiles.com/fairseq/models/mbart50/mbart50.ft.n1.tar.gz

**Comment by patil-suraj** [comment]
It's not a bug, sorry about the wrong label.

**Comment by raymondhs** [comment]
Hello, is there any update on this issue? I was unable to download the many-to-one model. Thanks.
